specification/vmware/resource-manager/Microsoft.AVS/stable/2020-03-20/examples/PrivateClouds_CreateOrUpdate.json

```diff
@@ -0,0 +1,119 @@
+{
+  "parameters": {
+    "api-version": "2020-03-20",
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "group1",
+    "privateCloudName": "cloud1",
+    "privateCloud": {
+      "location": "eastus2",
+      "sku": {
+        "name": "AV36"
```

Comment:
Is it possible to change to "av36'? Because in response it is "av36" after testing.